app/src/main/java/com/SecUpwN/AIMSICD/AIMSICD.java

```diff
@@ -93,7 +93,6 @@
     private ActionBarDrawerToggle mDrawerToggle;
     private CharSequence mDrawerTitle;
     private CharSequence mTitle;
-    public static ProgressBar mProgressBar;
```

Comment:
I guess so. But why fully remove it instead of replacing it with soething like [SmoothProgressBar](https://github.com/castorflex/SmoothProgressBar)?